site2/docs/client-libraries-python.md

```diff
@@ -9,29 +9,21 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ````
 
+You can use a Pulsar Python client to create producers, consumers, and readers.
 
-Pulsar Python client library is a wrapper over the existing [C++ client library](client-libraries-cpp.md) and exposes all of the [same features](/api/cpp). You can find the code in the [Python directory](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/python) of the C++ client code.
```

Comment:
Why removing `Pulsar Python client library is a wrapper over the existing [C++ client library](client-libraries-cpp.md).`? Does it provide more context for users?

Reply:
I formerly move it to the build from source section since users need to know this technical detail there.

Later I remove the build from source section as described in https://github.com/apache/pulsar/pull/17796#discussion_r977310599.

This statement can be removed:

1. Users don't need to know the Python client is a wrapper of C++ client as they just use it.
2. It's not quite correct that we _can_ provide the same features. Some gule code is still required.